docs/tutorial/notebook/reading_a_config_file.ipynb

```diff
@@ -0,0 +1,608 @@
+{
```

Comment:
<p>Change the goal of this tutorial to:</p><p><strong>Controlling training and inference with config files</strong></p><p> </p><p>The tutorial should answer these questions for a user:</p><ul><li>How do I specify where my data is?</li><li>How do I check the current training or inference parameters? How do I change them?</li><li>How do command line arguments and config file parameters interact? When should either be used and which has precedence?</li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/isl-org/Open3D-ML/pull/512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p><strong>To Do </strong>- Sanskar</p>

Reply:
Adding information about command line arguments in this file doesn't make sense as this tutorial is intended for reading config file through python code, and we don't talk about our launch script anywhere.

I am planning to add a new tutorial on using our launch script `scripts/run_pipeline.py`, which will include the usage of command line arguments, precedences, distributed training etc..